pages/measurement-sets/[id].js

```diff
@@ -266,6 +226,18 @@ export default function MeasurementSet({
               sequencingPlatforms={sequencingPlatforms}
             />
           )}
+          {controlFileSets?.length > 0 && (
```

Comment:
`controlFileSets` is required, so you don’t need the question mark. If none exist, you still have an empty array that works without the question mark.

Reply:
done